src/main/java/com/sourcery/gymapp/backend/workout/service/RoutineService.java

```diff
@@ -0,0 +1,80 @@
+package com.sourcery.gymapp.backend.workout.service;
+
+import com.sourcery.gymapp.backend.workout.dto.CreateRoutineDto;
+import com.sourcery.gymapp.backend.workout.dto.ResponseRoutineDto;
+import com.sourcery.gymapp.backend.workout.exception.RoutineNotFoundException;
+import com.sourcery.gymapp.backend.workout.exception.UserNotFoundException;
+import com.sourcery.gymapp.backend.workout.mapper.RoutineMapper;
+import com.sourcery.gymapp.backend.workout.model.Routine;
+import com.sourcery.gymapp.backend.workout.repository.RoutineRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.AuditorAware;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class RoutineService {
+    private final RoutineRepository routineRepository;
+    private final RoutineMapper routineMapper;
+    private final AuditorAware<UUID> auditorAware;
```

Comment:
we should use current user service to get current user id
(single responsibility principle -> auditor is responsible for auditing only)

Reply:
we don't have the current user service yet

Reply:
do you want to just create a CurrentUserService to get the auditorAware there for the future?

Reply:
add // TODO: replace with CurrentUserService then

Reply:
Or even better to create CurrentUserService (within the module) and hardcode some type of users UUID there